R-package/R/lgb.convert.R

```diff
@@ -1,11 +1,13 @@
-#' @name lgb.prepare2
+#' @name lgb.convert
 #' @title Data preparator for LightGBM datasets (integer)
 #' @description Attempts to prepare a clean dataset to prepare to put in a \code{lgb.Dataset}.
-#'              Factors and characters are converted to numeric (specifically: integer).
-#'              Please use \code{\link{lgb.prepare_rules2}} if you want to apply this transformation to
+#'              Factors and characters are converted to integer.
+#'              Please use \code{\link{lgb.convert_with_rules}} if you want to apply this transformation to
 #'              other datasets. This is useful if you have a specific need for integer dataset instead
 #'              of numeric dataset. Note that there are programs which do not support integer-only
 #'              input. Consider this as a half memory technique which is dangerous, especially for LightGBM.
```

Comment:
Remove these two lines as it's obsolete since https://github.com/microsoft/LightGBM/issues/3094 is fixed:

```
Note that there are programs which do not support integer-only
input. Consider this as a half memory technique which is dangerous, especially for LightGBM.
```

Reply:
oh yeah!! Thank you

Reply:
updated! https://github.com/microsoft/LightGBM/pull/3095/commits/d023d94763582212546574526606e49cd902efa9